assets/armsnippets.jsonc

```diff
@@ -140,6 +140,120 @@
         ],
         "description": "Application Service Plan (Server Farm)"
     },
+    "Application Gateway": {
+        "prefix": "arm-app-gw",

```

Comment:
gw > gateway.

This war, when typing gateway, the snippet will be presented and also brings in line with other resource types with gateway in the name.

![image](https://user-images.githubusercontent.com/7844635/77958524-04cfc500-728a-11ea-84bd-40d00693517a.png)